app/src/view/screens/BottomTab/Categories.js

```
@@ -1462,231 +1524,247 @@ export default function Categories(identifier) {
+                    ) : (
+                      <FlatList
+                        style={{ margin: 8, flex: 1 }}
+                        //contentContainerStyle={{marginBottom:hp(5)}}
+                        showsVerticalScrollIndicator={false}
+                        data={topData}
+                        //keyExtractor={item => item.id.toString()}
+                        numColumns={3} // Set the number of columns to 3
+                        renderItem={({ item }) => renderAvailableApps(item)}
+                      />
+                    )}
+                  </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)